manifests/plugin/dynflow/params.pp

```diff
@@ -0,0 +1,8 @@
+# Default parameters for the Dynflow smart proxy plugin
+class foreman_proxy::plugin::dynflow::params {
+  include ::foreman_proxy
```

Comment:
From #192 this can trigger weird issues with kafo. Maybe group should default to undef instead?